app/views/shared/_navbar.html.erb

```diff
@@ -5,13 +5,16 @@
   </div>
   <ul class="nav navbar-nav navbar-right">
     <li>
+      <%= link_to "Started Applications: #{@bucket.total_jobs}", bucket_index_path %>
```

Comment:
@jmejia: I'm not sure about this one. Since the link is "Started Applications: 1". I was using string interpolation to get the number of applications in the bucket. Without the string interpolation I can't get the number or I would need to get rid of the string "Started Applications"